src/FileResolver.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SensioLabs\Deptrac;
+
+use SensioLabs\Deptrac\Configuration\Configuration;
+use Symfony\Component\Finder\Finder;
+
+class FileResolver
+{
+    /**
+     * @param Configuration $configuration
+     *
+     * @throws \InvalidArgumentException
```

Comment:
When will this exception ever be thrown?

Reply:
`Finder::in()`-Method will throw this exception if a directory does not exists